compiler/src/dotty/tools/dotc/printing/RefinedPrinter.scala

```diff
@@ -458,7 +458,7 @@ class RefinedPrinter(_ctx: Context) extends PlainPrinter(_ctx) {
               	// avoid the double `*` in this case.
               	toText(expr)
               case _ =>
-                toText(expr) ~ "*"
```

Comment:
`*` instead of `_: *` is valid syntax in Scala 3.

Reply:
Yes, it should be "*",  not ": *"